cloud_sql/mysql_instance_backupdr_backup/main.tf

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# [START cloud_sql_mysql_instance_backupdr_backup]
+resource "google_sql_database_instance" "instance" {
+  name             = "mysql-instance-backupdr-backup"
+  region           = "asia-northeast1"
+  database_version = "MYSQL_8_0"
+  settings {
+    tier = "db-f1-micro"
+    backup_configuration {
+      enabled            = true
+      binary_log_enabled = true
+      start_time         = "20:55"
+    }
+  }
+  backupdr_backup = "projects/<project-id>/locations/asia-northeast1/backupVaults/<backup-vault-id>/dataSources/<data-source-id>/backups/<backup-id>"
+  # set `deletion_protection` to true, will ensure that one cannot accidentally delete this instance by
+  # use of Terraform whereas `deletion_protection_enabled` flag protects this instance at the GCP level.
+  deletion_protection = false
```

Comment:
Since you have set these samples as undertested (through the inclusion of the test.yaml file), this deletion protection is not required for CI purposes. Please remove it from this and other samples.